samples/military_feature_schemas/Fields_Cyberspace.csv

```diff
@@ -3,6 +3,7 @@ identity,Integer,4,Standard Identity,NULLABLE,Identity,,False,,
 context,SmallInteger,2,Context,NULLABLE,Context,,False,,
 symbolset,SmallInteger,2,Symbol Set,NON_NULLABLE,,60,False,,
 symbolentity,Integer,4,Cyberspace Entity,NON_NULLABLE,Cyberspace_Entities,,False,,
+status,SmallInteger,2,Status,NULLABLE,Statuses,,False,,
```

Comment:
Status has been removed from Cyberspace now, in PR #247   

I concur that a feature class should either have an operationalcondition field or a status field but not both.

The question now is:

1. Is it better to have two different field names using two different domain names, where some feature classes use status and some use operationalcondition? ... or ...

2. Is it better to settle on a single field name used in all the affected feature classes, but retain two different domains (status vs. operationalcondition), and decide which of the two domains is best with any given feature class?

I prefer the second option, because the operationalcondition values imply status as well.  So I would suggest having just a single "status" field and using the "status" domain with it for some feature classes and using the "operationalcondition" domain with it for other feature classes.